build.sbt

```diff
@@ -54,9 +54,9 @@ libraryDependencies ++= Seq(
   "javax.servlet"                   %  "javax.servlet-api"            % "3.1.0"          % "provided",
   "junit"                           %  "junit"                        % "4.12"           % "test",
   "org.scalatra"                    %% "scalatra-scalatest"           % ScalatraVersion  % "test",
-  "org.mockito"                     %  "mockito-core"                 % "2.7.16"         % "test",
+  "org.mockito"                     %  "mockito-core"                 % "2.7.22"         % "test",
   "com.wix"                         %  "wix-embedded-mysql"           % "2.1.4"          % "test",
-  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.14"           % "test"
+  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.23"           % "test"
```

Comment:
> I raised an issue yandex-qatools/postgresql-embedded#76 but so far no answer/solution.

I've found that issue. Good job 👍 

> Not sure why GitBucket must work with Java 9

Some users are already trying to use GitBucket with Java 9. I'm not daily testing GitBucket with Java 9, but I would like to run test cases on Travis at least.